mailbox/store/src/main/java/org/apache/james/mailbox/store/search/LazyMessageSearchIndex.java

```diff
@@ -1,147 +1,141 @@
-/****************************************************************

```

Comment:
This changeset is not readable. You should have a `change line delimiters for LazyMessageSearchIndex` commit...